avm/src/vm/opcode.rs

```diff
@@ -1617,7 +1612,7 @@ pub static OPCODES: Lazy<HashMap<i64, ByteOpcode>> = Lazy::new(|| {
     }).collect();
     hand_mem.write_fractal(args[2], &Vec::new());
     for mem in output {
-      HandlerMemory::transfer(&mem, CLOSURE_ARG_MEM_START, hand_mem, CLOSURE_ARG_MEM_START);
```

Comment:
do we still need `transfer`?

Reply:
It's used by the HTTP and Datastore opcodes, and in those cases we really do need a copy and we really do want just the relevant data instead of wholesale swiping of memory blocks, so we do still need it.